# Fix makefile to use EB values
# Jordi Blasco <jordi.blasco@hpcnow.com>
--- MoreRONN-4.9.orig/src/Makefile.am	2019-03-19 05:34:06.000000000 +0000
+++ MoreRONN-4.9/src/Makefile.am	2020-03-23 04:31:32.250734473 +0000
@@ -7,9 +7,8 @@
 moreRONN_LDADD = -lrt -lm -lgomp
 
 install-exec-hook:
-	ln -fs $(prefix)/bin/moreRONN /usr/local/bin/moreRONN_49
+	ln -fs $(prefix)/bin/moreRONN $(prefix)/bin/moreRONN_49
 	cp ../data/* $(prefix)
 
 uninstall-hook:
 	rm -rf $(prefix)
-	rm -f /usr/local/bin/moreRONN_49
